proposals/lip-0047.md

```diff
@@ -380,7 +379,6 @@ A transaction with module name `MODULE_NAME_POA` and command name `COMMAND_UPDAT
 ```python
 def verify(trs: Transaction) -> None:
     trsParams = decode(updateGeneratorKeyParamsSchema, trs.params)
```

Comment:
Here I think we could even remove this line, since `trsParams` is not used in the function.